setup.py

```diff
@@ -3,25 +3,36 @@
 from io import open
 import os
 import sys
-
+import platform
+from setuptools import Extension, setup
+from Cython.Build import cythonize
 
 here = os.path.abspath(os.path.dirname(__file__))
 
 # Get the long description from the README file
 with open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
-cmake_args = []
-for option in ("CYTHON_LINETRACE", "CYTHON_ANNOTATE"):
-    value = os.environ.get(option)
-    if value is not None and bool(int(value)):
-        cmake_args.append("-D{}:BOOL=TRUE".format(option))
-
-# On Mac, require OSX >= 10.9 so we can use te new libc++ targeting C++11
-if sys.platform == "darwin":
-    cmake_args.append("-DCMAKE_OSX_DEPLOYMENT_TARGET:STRING=10.9")
+extra_args = []
+linetrace = False
+if int(os.environ.get('CYTHON_LINETRACE', '0')):
+    linetrace = True
+    extra_args.append(('CYTHON_TRACE_NOGIL', '1'))
+
+extra_compile_args = []
+if platform.system() != 'Windows':
+    extra_compile_args.append('-std=c++11')
```

Comment:
what this means anyway? the issue is to make this work with older clang compilers preinstalled at macOS where they do support C++11 but aren't defaulting to it, Linux compilers (gcc/clang) were not the target but can benefit from it. This only isn't on Windows's cl.exe, msvc is targeting C++11 for a long time and harfbuzz doesn't support compile on older msvc compilers anyway, older mingw gcc compiler could benefit but detecting that is a bit tedious with low benefit so really not needed as all we wanted was to detect macos.